.github/workflows/build_test.yml

```diff
@@ -0,0 +1,44 @@
+---
+# vi: ts=2 sw=2 et:
+#
+name: Build test
+on:
+  pull_request:
+    paths:
+      - '**/meson.build'
+      - '.github/workflows/**'
+      - 'meson_options.txt'
+      - 'src/**'
+      - 'test/fuzz/**'
+      - 'travis-ci/managers/ubuntu-build-check.sh'
```

Comment:
Since the script is no longer compatible with Travis CI, I think it should be moved to ".github/workflows" and the line could be dropped.